lib/core/dispatchRequest.js

```diff
@@ -34,7 +34,8 @@ module.exports = function dispatchRequest(config) {
   config.headers = config.headers || {};
 
   // Transform request data
-  config.data = transformData(
+  config.data = transformData (
```

Comment:
format: style mismatch:
```suggestion
  config.data = transformData(
```

`16b835cf` ␣ at column 30 should be removed.